utils/logalloc.hh

```diff
@@ -685,16 +685,24 @@ struct reclaim_lock {
 // also allocate LSA memory. The object learns from failures how much it
 // should reserve up front in order to not cause allocation failures.
 class allocating_section {
-    size_t _lsa_reserve = 10; // in segments
-    size_t _std_reserve = 1024; // in bytes
+    // Do not decay below these minimal values
+    static constexpr size_t s_min_lsa_reserve = 10;
```

Comment:
Is there some new convention behind the `s_` name prefix (s = static)?

Reply:
Yes, not formalized.

Reply:
(and not new, see s_shard_id, s_rpc_client_idx_table)